test/core/renderControllerTests.ts

```diff
@@ -1,8 +1,80 @@
 ///<reference path="../testReference.ts" />
 
 describe("RenderController", () => {
-  // HACKHACK: #2083
-  it.skip("Components whose render() is triggered by another Component's render() will be drawn", () => {
+  let SVG_WIDTH = 400;
+  let SVG_HEIGHT = 300;
+
+  describe("configuring the render policy", () => {
+    let oldWarn: (msg: string) => void;
+    let storedWarningMsg: string;
+
+    before(() => {
+      oldWarn = Plottable.Utils.Window.warn;
+      Plottable.Utils.Window.warn = (warningMsg: string) => {
```

Comment:
The idea is to be able to assert something about the warning message, similar to [this best-practice](https://github.com/palantir/plottable/wiki/Testing-best-practices#assert-error-throwing-should-include-the-error-message) for `assert.throws()`.